Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:8.9.4
+
+LABEL description="Decrediton builder image"
+LABEL version="1.2"
+LABEL maintainer "john@netpurgatory.com"
+
+USER build
+WORKDIR /home/$USER
+
+COPY ./ /home/$USER
+
+RUN yarn install
```

Comment:
don't need to put install here, plus you need to `RUN yarn lint` and `RUN yarn test`